invenio_communities/templates/semantic-ui/invenio_communities/details/header.html

```diff
@@ -25,7 +25,7 @@
       </div>
       <div class="ten wide column">
           <h2 class="ui header">{{ community.metadata.title }}</h2>
-          <p>{{ community.metadata.description }}</p>
+          <p>{{ community.metadata.description|truncate(225) }}</p>
```

Comment:
is the description displayed anywhere besides the header in the community details page? if yes, then it is OK by me, but if not, then we are truncating the only place we display this field, and we should probably add somewhere below the full field display

Reply:
This is one of the two PRs that aim to truncate the text of the community metadata description, the other is linked to the linked issue. https://github.com/inveniosoftware/invenio-app-rdm/pull/1283

Reply:
sorry but I don't see how this answers my question